test/goaway_test.go

```diff
@@ -551,13 +550,15 @@ func (s) TestGoAwayThenClose(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error while listening. Err: %v", err)
 	}
-	s1 := grpc.NewServer()
-	defer s1.Stop()
-	ts := &funcServer{
-		unaryCall: func(context.Context, *testpb.SimpleRequest) (*testpb.SimpleResponse, error) {
+	//s1 := grpc.NewServer()
```

Comment:
why are these commented out? They should be removed?

Reply:
Done